lib/rufo/command.rb

```diff
@@ -82,7 +82,7 @@ def format_file(filename)
     code = File.read(filename)
 
     begin
-      result = format(code, File.dirname(filename))
+      result = format(code, @filename_for_dot_rufo || File.dirname(filename))
```

Comment:
@splattael testing it by exit status code is ok, but not enough I think. I would like to test it by mixing command_specs and rufo_spec (compare by file content).